src/components/Widgets/IndexUVI/IndexUVI.module.scss

```diff
@@ -0,0 +1,93 @@
+.wrapper {
+	border-radius: 20px;
+	padding: 30px;
+	padding-top: 5px;
+	text-align: center;
+	color: #000;
+	background-color: #f8f8f8;
+	box-shadow: 1px 1px 8px rgba(0, 0, 0, 0.493);
+	width: 60%;
+	height: 60%;
+	h3 {
+		margin-top: 0px;
+		font-size: 15px;
+		color: rgba(83, 83, 83, 0.685);
+	}
+	&__uvi {
+		position: relative;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		height: 100%;
+	}
+	svg {
+		transform: rotate(180deg);
+	}
+	path {
+		border: 1px solid #000;
+		fill: none;
+		stroke: url('#orange-to-pink');
+		stroke-linecap: round;
+		stroke-width: 8;
+		pointer-events: none;
+	}
+}
+.gradient {
+	position: relative;
+}
+
+.firstIndex {
```

Comment:
what the fuck?))))) do u know about nth-child?)

Reply:
done